package.json

```
@@ -30,10 +34,9 @@
     "recharts": "^1.8.5"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
```

to get this to run on heroku, we'll need to change this to:

```suggestion
    "start": "next start -p $PORT",
```